src/sentry/api/endpoints/organization_onboarding_tasks.py

```diff
@@ -4,8 +4,18 @@
 
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
-from sentry.models import OnboardingTaskStatus, OrganizationOnboardingTask
-from sentry.receivers import try_mark_onboarding_complete
+from sentry.models import OnboardingTaskStatus
+from sentry.onboarding_tasks import (
+    create_or_update_onboarding_task,
+    get_skippable_tasks,
+    get_status_lookup_by_key,
+    get_task_lookup_by_key,
+    try_mark_onboarding_complete,
+)
+
+TASK_LOOKUP_BY_KEY = get_task_lookup_by_key()
+SKIPPABLE_TASKS = get_skippable_tasks()
+STATUS_LOOKUP_BY_KEY = get_status_lookup_by_key()
```

Comment:
Nit: Wondering if it makes sense just to use these functions directly, since they're just reading these as static values off of the model.

Reply:
@wedamija I just did it this way to make the diff a little less bad